src/AddTodoForm.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+const AddTodoForm = (props) => {
+    const handleAddTodo = (event) => {
+        event.preventDefault();
+        const todoTitle = event.target.title.value;
+        event.target.reset();
+        props.onAddTodo(todoTitle); //newTodo??
+    };
```

Comment:
instead of using the DOM object, I would use a `title` state and do `setTitle("")` to reset it